sources/api/apiclient/Cargo.toml

```
@@ -39,11 +44,13 @@ serde_json.workspace = true
 signal-hook.workspace = true
 simplelog.workspace = true
 snafu = { workspace = true, features = ["futures"] }
+test-case = "*"
```

please use the test-case dependency in the workspace:

```suggestion
test-case.workspace = true
```

---

Fixed - switched dependencies